runtime/Cargo.toml

```diff
@@ -25,8 +25,8 @@ hex-literal = { optional = true, version = '0.3.1' }
 serde = { features = ['derive'], optional = true, version = '1.0.119' }
 
 # local dependencies
-pallet-account-linker = { git = "https://github.com/litentry/litentry-account-linker", package = "pallet-account-linker", default-features = false, branch = "dev", version = '0.0.1' }
-pallet-offchain-worker = { git = "https://github.com/litentry/litentry-offchain-worker", package = "pallet-offchain-worker", default-features = false, branch = "dev", version = '0.0.1' }
+pallet-account-linker = { git = "https://github.com/litentry/litentry-pallets", package = "pallet-account-linker", default-features = false, branch = "dev", version = '0.0.1' }
```

Comment:
Then we still need this version?

Reply:
Just put the version here as well, to have more control of versions. But actually we are not updating the version numbers in our pallets.